Mathlib/Data/Nat/Factorial/Basic.lean

```diff
@@ -30,7 +30,12 @@ def factorial : ℕ → ℕ
   | 0 => 1
   | succ n => succ n * factorial n
 
-/-- factorial notation `n!` -/
+/-- factorial notation `n!` for `Nat.factorial n`.
```

Comment:
Maybe:
```suggestion
/-- factorial notation `(n)!` for `Nat.factorial n`.
```
Perhaps also worth mentioning below that things like `0!` and `1!` work.